packages/remix-architect/server.ts

```diff
@@ -39,99 +48,26 @@ export function createRequestHandler({
   build,
   getLoadContext,
   mode = process.env.NODE_ENV,
+  apiGatewayVersion = APIGatewayVersion.v2
 }: {
   build: ServerBuild;
   getLoadContext?: GetLoadContextFunction;
   mode?: string;
+  apiGatewayVersion?: APIGatewayVersion;
```

Comment:
this would mean we use the api like this

```javascript
import * as build from '@remix-run/dev/server-build'
import { createRequestHandler } from '@sporza/architect'

export const handler = createRequestHandler({
  build,
  mode: process.env.NODE_ENV,
  apiGatewayVersion: 'v1' // or even APIGatewayVersion.v1 if you import the enum
})
```

Reply:
@mcansh wdyt, string `v1`/`v2` in favour of numbers `1`/`2` would be in line with how AWS documents the [new API Gateway](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/AWS_ApiGatewayV2.html)

Reply:
yup i agree